### DISCUSSION TITLE
Alignment in tablist.

### DISCUSSION THREAD

**Discussion by dmnchild** [Zrips/CMI#1903] [Feature Requests]
In setting up the tablist, it would be nice to somehow implement an alignment for a placeholder. 

ie I currently use:
    PlayerName: '%cmi_user_prefix% %cmi_user_display_name% &r%cmi_user_ping%'

I would love to be able to move the ping to the right, next to the connection graphic.
(possibly an option to enable/disable/replace the connection graphics with ping or any variant of infos for that)

**Comment by LogGits** [comment]
Just out of curiousity, couldnt u just set ur value to this?
`PlayerName: '%cmi_user_prefix% %cmi_user_display_name%&r      %cmi_user_ping%'`

**Comment by dmnchild** [comment]
It still just uses all the values and lines it up on the left. I do see you moved the &r for the display name, but cant imagine that will make the ping align to the right instead of being a space after the name. Either wa y, lack of options will keep me using TABList for now, but looking forward to more options being available for tab management in cmi.

**Comment by CloudeLecaw** [comment]
still would be nice to have. might so something like the %filler% in holograms?
